index.js

```diff
@@ -21,6 +22,19 @@ module.exports = (robot) => {
     // the whitelisted accounts.
     if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
       await apiForSheetsModule.checkClaStatus(context);
+      await checkPullRequestLabelsModule.checkChangelogLabel(context);
+    }
+  });
+
+  robot.on('pull_request.reopened', async context => {
+    if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
+      await checkPullRequestLabelsModule.checkChangelogLabel(context);

```

Comment:
Just one thought. Did you check that new contributors (who aren't members of the Oppia repo) are actually able to add labels to PRs?

Reply:
New contributors won't be able to add labels until they are on-board (as collaborators). Sandeep and I discussed this -- ideally, their onboarding mentors should help in putting up the labels.

Reply:
But then wouldn't their PRs immediately be closed as soon as they are opened?

Reply:
Let's not close the PR if the creator isn't a collaborator? Add "New-contributor" label or "Need to assign Changelog" label or add some message to indicate this? (not sure what to do if not closing the PR)

Reply:
In the future, we can assign it to the mentor and add a message asking the mentor to add the label.

Reply:
(Might need to add Project labels to the existing PRs, too)

Reply:
If you want to be safe, btw, maybe don't close the PR in general first? Just leave a message asking the author to please fix this?

Then if that's not working we can make this more stringent. But that may be a better incremental step.

If we go this route, maybe file an issue for Oppiabot to ensure that the changelog label is guaranteed to be there at time-of-merge.

Reply:
> If you want to be safe, btw, maybe don't close the PR in general first? Just leave a message asking the author to please fix this?
> Then if that's not working we can make this more stringent. But that may be a better incremental step.

I agree. This sounds much better for an incremental change.

> If we go this route, maybe file an issue for Oppiabot to ensure that the changelog label is guaranteed to be there at time-of-merge. (Perhaps it should be implemented as a repo check instead similar to travis-ci etc -- Oppiabot should be more of a helper rather than an enforcer.)

I will do this. Thanks!

Reply:
Filed #38 for this. Thanks!